components/org.wso2.carbon.identity.user.endpoint/src/main/java/org/wso2/carbon/identity/user/endpoint/impl/ValidateCodeApiServiceImpl.java

```diff
@@ -67,6 +69,6 @@ public Response validateCodePost(CodeValidationRequestDTO codeValidationRequestD
             Utils.handleInternalServerError(Constants.SERVER_ERROR,
                     IdentityRecoveryConstants.ErrorMessages.ERROR_CODE_UNEXPECTED.getCode(), LOG, throwable);
         }
-        return Response.accepted().build();
+        return Response.ok(Utils.getUserDTO(user)).build();
```

Comment:
Let's keep the same status codes.

Reply:
Fix in [95a1950](https://github.com/wso2-extensions/identity-governance/pull/463/commits/95a1950f71b5e77fe558482b010aee63d8b910ef)